project-base/scripts/install.sh

```diff
@@ -75,7 +75,7 @@ echo "Installing application inside a php-fpm container"
 
 docker-compose exec php-fpm composer install
 
-docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev
+docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev-quick error-pages-generate
```

Comment:
hello guys, here is change in project-base without added upgrade notes ...

Reply:
Wow! Thanks for pointing it out. I didn't realize this. 
I added the upgrade notes.